src/components/Project/Projects.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import GetProjects from "./ProjectsAPI.js";
+import Card from "./Card";
+import "./Projects.scss";
+
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+
+const useStyles = makeStyles({
+  table: {
+    minWidth: 200,
+    maxWidth: 1010,
+  },
+});
+
+function ProjectList() {
+  const classes = useStyles();
+  const [projects, setProjects] = useState([]);
+
+  const refresh = () => {
+    let list = [];
+    GetProjects()
+      .then((result) => {
+        list = result;
+      })
+      .then(() => setProjects(list));
+  };
+  useEffect(refresh, []);
```

Comment:
NABD but you're not reusing the refresh function so you can move all that code here